coap/adapter.go

```diff
@@ -135,10 +138,27 @@ func (svc *adapterService) Publish(ctx context.Context, token string, msg mainfl
 }
 
 func (svc *adapterService) Subscribe(chanID, subtopic, obsID string, o *Observer) error {
```

Comment:
why observer pointer?

Reply:
I don't know... I didn't modify this line